### PR TITLE
[LM-10471] Added Likes Screen and its Styles and Customizations 

### PR DESCRIPTION
**Pull request by sidcr7-likeminds**
## Description

**Features**

- Added Likes Screen
- Added styles related to Likes Screen
- Added customizations related to Likes Screen

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fix for breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [x] Description about the Feature/ Bugs added

### CODE REVIEW

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesFragment.kt [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = LmFeedFragmentLikesBinding.inflate(layoutInflater)
+
+        binding.apply {
+            customizeLikesFragmentHeaderView(headerViewLikes)
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesFragment.kt:51]: add todo for customizing list view

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesFragment.kt [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                        color
+                    )
+                )
+            }
+
+            return root
+        }
+    }
+
+    private fun customizeLikesFragmentHeaderView(headerViewLikes: LMFeedHeaderView) {
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesFragment.kt:69]: it should be `protected open fun` and add comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesListView.kt
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesListView.kt:1]: add comments to public functions

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/viewstyle/LMFeedLikesFragmentViewStyle.kt [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import android.text.TextUtils
+import androidx.annotation.ColorRes
+import com.likeminds.feed.android.core.R
+import com.likeminds.feed.android.core.ui.base.styles.LMFeedIconStyle
+import com.likeminds.feed.android.core.ui.base.styles.LMFeedTextStyle
+import com.likeminds.feed.android.core.ui.widgets.headerview.style.LMFeedHeaderViewStyle
+import com.likeminds.feed.android.core.ui.widgets.user.style.LMFeedUserViewStyle
+import com.likeminds.feed.android.core.utils.model.LMFeedPadding
+
+class LMFeedLikesFragmentViewStyle private constructor(
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/viewstyle/LMFeedLikesFragmentViewStyle.kt:12]: add styling comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/user/style/LMFeedUserViewStyle.kt [hunk 4]
```diff
@@ -0,0 +1,54 @@
+package com.likeminds.feed.android.core.ui.widgets.user.style
+
+import android.text.TextUtils
+import com.likeminds.feed.android.core.R
+import com.likeminds.feed.android.core.ui.base.styles.LMFeedImageStyle
+import com.likeminds.feed.android.core.ui.base.styles.LMFeedTextStyle
+import com.likeminds.feed.android.core.utils.LMFeedViewStyle
+
+class LMFeedUserViewStyle private constructor(
```
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/user/style/LMFeedUserViewStyle.kt:9]: add styling comments

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/user/view/LMFeedUserView.kt
Comment (ishaan1607) [inline likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/user/view/LMFeedUserView.kt:1]: add comments to public functions

**Review by ishaan1607 — CHANGES_REQUESTED** [review]

File: likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesFragment.kt (hunk 1 above)
Comment (sidcr7-likeminds) [reply likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/likes/view/LMFeedLikesFragment.kt:51]: added already in my notes

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**